Update mlnet cli docs (#18782)
dotnet/docs: docs/machine-learning/tutorials/sentiment-analysis-cli.md

## Pre-requisites

> - [.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1) or later

Reviewer: Is it 2.1 or 3.1?

    Now, you are ready to start using the CLI for this 'Sentiment Analysis' scenario.

    > [!NOTE]

>     > After finishing this tutorial you can also try with your own datasets as long as they are ready to be used for any of the ML tasks currently supported by the ML.NET CLI Preview which are *'Binary Classification', 'Classification', 'Regression',* and *'Recommendation'*.

Reviewer: ```suggestion
    > After finishing this tutorial you can also try with your own datasets as long as they are ready to be used for any of the ML tasks currently supported by the ML.NET CLI Preview which are *classification (binary and multiclass)*, *regression*, and *recommendation*.
```

## Run the 'mlnet classification' command

1. Run the following ML.NET CLI command:

    ```console
    mlnet classification --dataset "yelp_labelled.txt" --label-col 1 --has-header false --train-time 10
    ```

>     This command runs the **`mlnet classification` command**:

Reviewer: ```suggestion
    This command runs the `mlnet classification` command:
```

    - for the **ML task** of *classification*
    - uses the **dataset file `yelp_labelled.txt`** as training and testing dataset (internally the CLI will either use cross-validation or split it in two datasets, one for training and one for testing)
    - where the **objective/target column** you want to predict (commonly called **'label'**) is the **column with index 1** (that is the second column, since the index is zero-based)
    - does **not use a file header** with column names since this particular dataset file doesn't have a header
    - the **targeted exploration/train time** for the experiment is **10 seconds**

    You will see output from the CLI, similar to:

    <!-- markdownlint-disable MD023 MD025 -->

    # [Windows](#tab/windows)

    ![ML.NET CLI classification on PowerShell](./media/mlnet-cli/mlnet-classification-powershell.gif)

    # [macOS Bash](#tab/macosbash)

>     ![ML.NET CLI classification on Bash](./media/mlnet-cli/mlnet-auto-train-binary-classification-bash.gif)

Reviewer: the Mac GIF is outdated. I would suggest updating or removing it

    ```csharp
    static void Main(string[] args)
    {
        // Create single instance of sample data from first line of dataset for model input
        ModelInput sampleData = new ModelInput()
        {
            Col0 = @"Wow... Loved this place.",
        };

        // Make a single prediction on the sample data and print results
        var predictionResult = ConsumeModel.Predict(sampleData);

        Console.WriteLine("Using model to make single prediction -- Comparing actual Col1 with predicted Col1 from sample data...\n\n");
        Console.WriteLine($"Col0: {sampleData.Col0}");
        Console.WriteLine($"\n\nPredicted Col1 value {predictionResult.Prediction} \nPredicted Col1 scores: [{String.Join(",", predictionResult.Score)}]\n\n");
        Console.WriteLine("=============== End of process, hit any key to finish ===============");
        Console.ReadKey();
    }
    ```

>     - The first lines of code simply create a *single sample data*, in this case based on the first row of your dataset to be used for the prediction. You can also create you own 'hard-coded' data by updating the code:

Reviewer: ```suggestion
    - The first lines of code create a *single sample data*, in this case based on the first row of your dataset to be used for the prediction. You can also create you own 'hard-coded' data by updating the code:
```

        ```csharp
        ModelInput sampleData = new ModelInput()
        {
            Col0 = "The ML.NET CLI is great for getting started. Very cool!"
        };
        ```    

    - The next line of code uses the `ConsumeModel.Predict()` method on the specified input data to make a prediction and return the results (based on the ModelOutput.cs schema).
    
    - The last lines of code print out the proprties of the sample data (in this case the Comment) as well as the Sentiment prediction and corresponding Scores for positive sentiment (1) and negative sentiment (2).

1. Run the project, either using the original sample data loaded from the first row of the dataset or by providing your own custom hard-coded sample data. You should get a prediction comparable to:

    # [Windows](#tab/windows)

    Run the console app from Visual Studio by hitting F5 (Play button):

    ![ML.NET CLI run the app from Visual Studio](./media/mlnet-cli/mlnet-cli-console-app.png))

>     # [macOS Bash](#tab/macosbash)

Reviewer: The Mac image is not up do date, I would suggest updating or removing it

    Run the console app from the command-prompt by typing the following commands:

    ```dotnetcli
    cd SampleClassification
    cd SampleClassification.ConsoleApp

    dotnet run
    ```

>     ![ML.NET CLI run the app in bash](./media/mlnet-cli/sample-cli-prediction-execution-bash.png))

Reviewer: The Mac image is not up do date, I would suggest updating or removing it

Author: Yep that's the one TO DO for now. We haven't been able to get the current update of CLI working on Mac

> [!div class="checklist"]
>
> - Prepare your data for the selected ML task (problem to solve)
> - Run the 'mlnet classification' command in the CLI tool
> - Review the quality metric results
> - Understand the generated C# code to run the model (Code to use in your end-user app)

> > - Explore the generated C# code that was used to train the "best quality" model (earning purposes)

Reviewer: Is this right? earning? If not, I'd update or remove this part

Author: Not sure, this bit of text was there before I think. Must be learning purposes
